css-fonts/styles.css

```diff
@@ -0,0 +1,8 @@
+head{
+  font-family: 'Yaldevi', sans-serif;
+}
```

Comment:
The `<head>` element doesn't contain anything that is directly visible on the page, so setting a `font-family` on it won't do anything. Please update your stylesheet so that the user can see both fonts being used on the page.